libsolidity/formal/CHC.cpp

```diff
@@ -79,6 +79,26 @@ void CHC::analyze(SourceUnit const& _source)
 		source->accept(*this);
 
 	checkVerificationTargets();
+
+	bool ranSolver = true;
+#ifndef HAVE_Z3
+	auto solver = dynamic_cast<CHCSmtLib2Interface const*>(m_interface.get());
```

Comment:
```suggestion
	auto const* solver = dynamic_cast<CHCSmtLib2Interface const*>(m_interface.get());
```

Reply:
Ah this was actually debug leftover, removed